src/Math/makefile

```diff
@@ -5,10 +5,13 @@ endif
 ifndef ($(VARIANT))
 VARIANT             = release
 endif
+ifndef BUILD_DIR
```

Comment:
All this is Linux-related, Windows does not use makefile, actually this is something else todo on Windows is to make the library compile.